sphinxcontrib/openapi/openapi30.py

```diff
@@ -262,6 +263,27 @@ def _httpresource(endpoint, method, properties, render_examples):
         for line in param.get('description', '').splitlines():
             yield '{indent}{indent}{line}'.format(**locals())
 
+    # print request content
+    if render_request:
```

Comment:
It looks like a functionality duplication. How does this differs from `render_examples` below?

Reply:
It's an untouched copy of a request body from the Spec file. Developers need to know the exact specification of a request.